code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1936,11 +1936,23 @@
 		return T
 	else
 		var/mob/living/simple_animal/hostile/necro/zombie/turned/T = new(get_turf(src), master, (retain_mind ? src : null))
+		T.add_spell(/spell/aoe_turf/necro/zombie/evolve)
+		if(isgrey(src))
+			T.icon_state = "mauled_laborer"
```

Comment:
fun names but why not just rename the sprites to be "zombie_grey" "zombie_vox" etc then do icon_state = "zombie_" + species.name

Just a style preference thing but it saves you 8 lines, a bunch of paste, and it's more maintainable for future races added

Reply:
Right now, these are just placeholder sprites

Reply:
Do this change since you aren't updating sprites

Reply:
These are actual sprites, but I'm just using them for other purposes...